### PR TITLE
fix: Move disabled connectors at bottom

### PR DESCRIPTION
**Pull request by susrithasabbini**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Moved all the disabled connectors to the bottom of the table

## Motivation and Context

Can differentiate between enabled and disabled connectors easily at top and bottom of the table.

## Where to test it?

- [x] INTEG
- [x] SANDBOX
- [ ] PROD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/screens/Connectors/ConnectorList.res [hunk 1]
```diff
@@ -68,6 +71,8 @@ let make = (~isPayoutFlow=false) => {
     ? payoutConnectorList
     : connectorList
 
+  // Console.log(configuredConnectors)
```
Comment (kanikabansal-juspay) [inline src/screens/Connectors/ConnectorList.res:74]: You can remove this comment

Comment (susrithasabbini) [reply src/screens/Connectors/ConnectorList.res]: Done

File: src/screens/HSwitchUtils.res [hunk 2]
```diff
@@ -248,3 +248,12 @@ let getConnectorIDFromUrl = (urlList, defaultValue) => {
   | _ => urlList->Array.get(1)->Option.getOr(defaultValue)
   }
 }
+
+let sortByDisableField = (arr: array<'a>, getDisabledStatus: 'a => bool) => {
```
Comment (JeevaRamu0104) [inline src/screens/HSwitchUtils.res:252]: Please write this in ConnectorUtils file as this is specific to Connector

File: src/screens/Connectors/FraudAndRisk/FRMSelect.res [hunk 3]
```diff
@@ -117,6 +117,7 @@ let make = () => {
         let previousData = frmList->Array.map(ConnectorListMapper.getProcessorPayloadType)
         setFilteredFRMData(_ => previousData->Array.map(Nullable.make))
         setPreviouslyConnectedData(_ => previousData->Array.map(Nullable.make))
+        ConnectorUtils.sortByDisableField(previousData, p => p.disabled)
```
Comment (Riddhiagrawal001) [inline src/screens/Connectors/FraudAndRisk/FRMSelect.res:120]: Can you provide a better variable name than `p` and `c`

**Review by Riddhiagrawal001 — APPROVED** [review]

**Review by kanikabansal-juspay — APPROVED** [review]

**Review by JeevaRamu0104 — APPROVED** [review]

**Pull request merged**